openwisp_radius/integrations/monitoring/apps.py

```diff
@@ -0,0 +1,121 @@
+from django.apps import AppConfig
+from django.db import models
+from django.db.models import Count, Sum
+from django.db.models.functions import Cast, Round
+from django.db.models.signals import post_save
+from django.utils.timezone import localdate
+from django.utils.translation import gettext_lazy as _
+from openwisp_monitoring.monitoring.configuration import (
+    _register_chart_configuration_choice,
+    register_metric,
+)
+from swapper import load_model
+
+from openwisp_utils.admin_theme import register_dashboard_chart
+
+from .utils import get_datetime_filter_start_date, get_datetime_filter_stop_date
+
+
+class OpenwispRadiusMonitoringConfig(AppConfig):
+    name = 'openwisp_radius.integrations.monitoring'
+    label = 'openwisp_radius_monitoring'
+    verbose_name = _('OpenWISP RADIUS Monitoring')
+
+    def ready(self):
+        super().ready()
+        self.register_dashboard_charts()
+        self.register_radius_metrics()
+        self.connect_signal_receivers()
+
+    def register_dashboard_charts(self):
+        register_dashboard_chart(
+            position=30,
+            config={
+                'name': _("Today's RADIUS sessions"),
```

Comment:
quoting style

Reply:
The double quotes is used because there's an apostrophe present in the text. Shall change it to 
```suggestion
                'name': _('Today\'s RADIUS sessions'),
```

IMO,  this is less readable